### ISSUE TITLE
Low training loss, but low map in coco when using transfer lerning?

### ISSUE THREAD

**Issue by Edwardmark** [ultralytics/yolov5#3496]
I train a rpn using coco by setting single-cls to true, and I try to fine-tune the network with num_class=80, with coco, the training loss is good, but the validataion map is quite low, I am really confused.  I freeze all layers using the following code:
```
freeze = ['model.%s.' % x for x in range(24)] # freeze backbone layer
```
Why is that?
The logs is as follows:
![image](https://user-images.githubusercontent.com/15247551/120955616-57591b00-c784-11eb-9c0e-e50743a22399.png)
But when I do not use freeze, the result is normal as follows:
![image](https://user-images.githubusercontent.com/15247551/120964329-c1c68700-c795-11eb-85d9-7f4372f78c48.png)

**Comment by Edwardmark** [comment]
@glenn-jocher Thanks in advance, could you please give me some guide why is that? It is so strange.

**Comment by glenn-jocher** [comment]
@Edwardmark if you change a setting and produce worse results after your change, then your experiment has simply not produced an improvement. There's nothing strange about it.

**Comment by glenn-jocher** [comment]
@Edwardmark you may also want to see **Transfer Learning with Frozen Layers** tutorial below.

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by Edwardmark** [comment]
@glenn-jocher That is not what I mean. The strange thing is the training loss is low, but the evalutaion map is bad nearly zero.

**Comment by Edwardmark** [comment]
@glenn-jocher If using freezing all layers, the loss is low as 0.1, but map is low as 0.003; when not using freezing layers, the loss is high as 0.25, but map is high as 0.275. So I was wondering, is something I misunderstood. And I have read the tutorials, and just followed the guide.

**Comment by glenn-jocher** [comment]
@Edwardmark I think the problem is that you are making assumptions based upon some sort of intuition you think you have regarding loss magnitudes at different training configurations. Unless you have previous empirical results demonstrating problems in the current implementation then you should simply learn from this and future experiments.
